exo-deploy/terraform/aws/network/bastion/bastion.tf

```diff
@@ -0,0 +1,32 @@
+module "ami" {
```

Comment:
I don't feel strongly about this, but consistency would be nice. The segmentio stack uses ubuntu-based AMIs instead of Amazon's ECS AMIs